.gitignore

```diff
@@ -47,29 +47,8 @@ buck-out/
 !.vscode/launch.json
 !.vscode/extensions.json
 
-#-------------------------------
-# RENATIVE
-#-------------------------------
-
-platformBuilds/
-platformAssets/
-package-lock.json
```

Comment:
Is this one deliberately removed?

Reply:
yarp 👍

Reply:
But so should it now also be part of the repo then? Looking at; https://github.com/pavjacko/renative/pull/439/files#diff-32607347f8126e6534ebc7ebaec4853d

Reply:
Ooops yeah, [it was in there twice](https://github.com/pavjacko/renative/blob/825e7ac258b9c18d064265508f7885e6130ad8dd/.gitignore#L33)